src/__tests__/patients/search/NoPatientsExist.test.tsx

```diff
@@ -1,33 +1,18 @@
-import { Icon, Typography, Button } from '@hospitalrun/components'
-import { mount } from 'enzyme'
+import { render, screen } from '@testing-library/react'
 import React from 'react'
 
 import NoPatientsExist from '../../../patients/search/NoPatientsExist'
 
 describe('NoPatientsExist', () => {
-  const setup = () => mount(<NoPatientsExist />)
+  const setup = () => render(<NoPatientsExist />)
 
   it('should render an icon and a button with typography', () => {
-    const wrapper = setup()
+    setup()
 
-    const addNewPatient = wrapper.find(NoPatientsExist)
-    expect(addNewPatient).toHaveLength(1)
+    expect(screen.getByRole('heading')).toBeInTheDocument()
+    expect(screen.getByText('patients.noPatients')).toBeInTheDocument()
 
-    const icon = wrapper.find(Icon).first()
-    const typography = wrapper.find(Typography)
-    const button = wrapper.find(Button)
-    const iconType = icon.prop('icon')
-    const iconSize = icon.prop('size')
-    const typographyText = typography.prop('children')
-    const typographyVariant = typography.prop('variant')
-    const buttonIcon = button.prop('icon')
-    const buttonText = button.prop('children')
-
-    expect(iconType).toEqual('patients')
-    expect(iconSize).toEqual('6x')
-    expect(typographyText).toEqual('patients.noPatients')
-    expect(typographyVariant).toEqual('h5')
```

Comment:
h5 is implementation detail isn't it?
If someone changed it to h6 or h4 do we want to break test?

Reply:
I'd try using a `getByRole('heading', { name: /xxx/i })`

Reply:
Added `expect(screen.getByRole('heading', { name: /patients.noPatients/i })).toBeInTheDocument()`